tests/TestCase.php

```diff
@@ -56,8 +56,6 @@ protected function getEnvironmentSetUp($app)
         $app['config']->set('database.connections.mysql', $config['connections']['mysql']);
         $app['config']->set('database.connections.mongodb', $config['connections']['mongodb']);
         $app['config']->set('database.connections.mongodb2', $config['connections']['mongodb']);
```

Comment:
Are `database.connections.mysql` and `database.connections.mongodb2` actually used for anything?